docs/plugin-proposal-private-property-in-object.md

```diff
@@ -0,0 +1,123 @@
+---
+id: babel-plugin-proposal-private-property-in-object
+title: @babel/plugin-proposal-private-property-in-object
+sidebar_label: proposal-private-property-in-object
+---
+
+## Example
+
+**In**
+
+```javascript
+class Foo {
+  #bar = "bar";
+
+  test(obj) {
+    return #bar in obj;
+  }
+}
+```
+
+**Out**
+
+```javascript
+class Foo {
+  constructor() {
+    _bar.set(this, {
+      writable: true,
+      value: "bar"
+    });
+  }
+
+  test() {
+    return _bar.has(this);
+  }
+
+}
+
+var _bar = new WeakMap();
+```
+
+## Installation
+
+```sh
+npm install --save-dev @babel/plugin-property-private-property-in-object
+```
+
+## Usage
+
+### With a configuration file (Recommended)
+
+```json
+{
+  "plugins": ["@babel/plugin-property-private-property-in-object
+}
+```
+
+### Via CLI
+
+```sh
+babel --plugins @babel/plugin-property-private-property-in-object
+```
+
+### Via Node API
+
+```javascript
+require("@babel/core").transform("code", {
+  plugins: ["@babel/plugin-property-private-property-in-object
+});
+```
+
+## Options
+
+### `loose`
+
+`boolean`, defaults to `false`.
+
+> Note: The `loose` mode configuration setting _must_ be the same as [`@babel/proposal-class-properties`](plugin-proposal-class-properties.md).
+
+When true, private property `in` expressions will check for a own property on the object, instead of checking for presence inside a `WeakSet`. This results in improved
```

Comment:
```suggestion
When true, private property `in` expressions will check own properties (as opposed to inherited ones) on the object, instead of checking for presence inside a `WeakSet`. This results in improved
```